packages/backend/src/controllers/restaurants.test.js

```diff
@@ -401,6 +401,8 @@ describe('when logged in', () => {
       categories: [],
       address: 'Senaatintori',
       coordinates: { latitude: 60, longitude: 24 },
+      notSelectedAmount: 0,
+      resultAmount: 0,
```

Comment:
No tests or documentation for the new endpoints?

Reply:
Added tests and documentation to new endpoints.